crates/query-engine/sql/src/sql/ast.rs

```diff
@@ -164,12 +164,6 @@ pub enum Expression {
     },
     /// NOT clause
     Not(Box<Expression>),
-    /// A binary operation on two scalar expression
```

Comment:
Removed in favour of `FunctionCall`

Reply:
Hey, I think it would actually be nice to keep BinaryOperation as well, because essentially we end up have two different implementations in convert.rs with an `if`, so I think it makes more sense to just have two structures. That way we also:
1. mimic the postgres ast as it is
2. we make sure that we build binary operations with exactly two arguments, reducing potential errors with generated queries
3. avoid branching in favor of preserving structures

Reply:
@plcplc